packages/common/src/api/tan-query/coins/useArtistCoinMembers.ts

```diff
@@ -80,17 +97,27 @@ export const useArtistCoinMembers = (
 
       const response = await sdk.coins.getCoinMembers(params)
 
-      const members: CoinMember[] = (response.data ?? []).map(
-        (member: any) => ({
-          user_id: decodeHashId(member.userId) ?? 0,
-          balance: member.balance
-        })
-      )
+      const members = (response.data ?? []).map((member) => {
+        const decimals = artistCoin?.decimals
+        const balanceFD = new FixedDecimal(
+          BigInt(member.balance.toString()),
+          decimals
+        )
+
+        return {
+          userId: decodeHashId(member.userId) ?? 0,
+          balance: member.balance,
+          balanceLocaleString: balanceFD.toLocaleString('en-US', {
```

Comment:
feel like we should probably be using `packages/common/src/store/ui/buy-sell/useTokenAmountFormatting.ts` here?